rendering/glb.ksy

```diff
@@ -0,0 +1,69 @@
+meta:
+  id: glb
+  title: GL Transmission Format, binary container
+  file-extension: glb
+  endian: le
+  license: MIT
+  xref:
+    mime: model/gltf-binary
+    wikidata: Q28135989
```

Comment:
Let's reorder the keys per the recommendation in the [style guide](https://doc.kaitai.io/ksy_style_guide.html#meta):

```suggestion
  title: GL Transmission Format, binary container
  file-extension: glb
  xref:
    mime: model/gltf-binary
    wikidata: Q28135989
  license: MIT
  endian: le
```